jira.go

```diff
@@ -117,16 +124,22 @@ func openJiraTicket(flags flags, projectInfo jsn.Json, vulnForJira interface{},
 
 	if flags.optionalFlags.priorityIsSeverity {
 		var priority PriorityType
-		jiraMappingEnvVarName := fmt.Sprintf("SNYK_JIRA_PRIORITY_FOR_%s_VULN", strings.ToUpper(jsonVuln.K("issueData").K("severity").String().Value))
+
+		severity := jsonVuln.K("issueData").K("severity").String().Value
```

Comment:
wouldn't that fail if this is a code project first since the key issueData doesn't exist? or json.K("something") returns "" if it can't find the key?